### PR TITLE
feat(journaldreceiver): add missing dependencies

### PR DESCRIPTION
**Pull request by Wolodija**
It resolves the following error:

```
otelcol_1           | 2022-06-02T09:28:12.441Z  info    builder/exporters_builder.go:255        Exporter was built.     {"kind": "exporter", "name": "logging"}
otelcol_1           | 2022-06-02T09:28:12.441Z  info    builder/pipelines_builder.go:224        Pipeline was built.     {"kind": "pipeline", "name": "logs"}
otelcol_1           | 2022-06-02T09:28:12.442Z  info    builder/receivers_builder.go:226        Receiver was built.     {"kind": "receiver", "name": "journald", "datatype": "logs"}
otelcol_1           | 2022-06-02T09:28:12.442Z  info    service/telemetry.go:109        Setting up own telemetry...
otelcol_1           | 2022-06-02T09:28:12.465Z  info    service/telemetry.go:129        Serving Prometheus metrics      {"address": ":8888", "level": "basic", "service.instance.id": "dad331ee-30d4-47f1-9511-ce068fa5204f", "service.version": "latest"}
otelcol_1           | 2022-06-02T09:28:12.465Z  info    service/service.go:76   Starting extensions...
otelcol_1           | 2022-06-02T09:28:12.465Z  info    service/service.go:81   Starting exporters...
otelcol_1           | 2022-06-02T09:28:12.465Z  info    builder/exporters_builder.go:40 Exporter is starting... {"kind": "exporter", "name": "logging"}
otelcol_1           | 2022-06-02T09:28:12.465Z  info    builder/exporters_builder.go:48 Exporter started.       {"kind": "exporter", "name": "logging"}
otelcol_1           | 2022-06-02T09:28:12.465Z  info    service/service.go:86   Starting processors...
otelcol_1           | 2022-06-02T09:28:12.465Z  info    builder/pipelines_builder.go:54 Pipeline is starting... {"kind": "pipeline", "name": "logs"}
otelcol_1           | 2022-06-02T09:28:12.465Z  info    builder/pipelines_builder.go:65 Pipeline is started.    {"kind": "pipeline", "name": "logs"}
otelcol_1           | 2022-06-02T09:28:12.465Z  info    service/service.go:91   Starting receivers...
otelcol_1           | 2022-06-02T09:28:12.465Z  info    builder/receivers_builder.go:68 Receiver is starting... {"kind": "receiver", "name": "journald"}
otelcol_1           | 2022-06-02T09:28:12.465Z  info    stanza@v0.51.0/receiver.go:54   Starting stanza receiver        {"kind": "receiver", "name": "journald"}
otelcol_1           | Error: cannot start receivers: start stanza: start journalctl: exec: "journalctl": executable file not found in $PATH
otelcol_1           | 2022/06/02 09:28:12 collector server run finished with error: cannot start receivers: start stanza: start journalctl: exec: "journalctl": executable file not found in $PATH
2022-02-07-otc-systemd_otelcol_1 exited with code 1
```

### CODE REVIEW

**Review by kkujawa-sumo — COMMENTED** [review]
What error is observed without these dependencies?

**Comment by sumo-drosiek** [comment]
I updated description

**Review by kkujawa-sumo — APPROVED** [review]

**Comment by swiatekm-sumo** [comment]
How much does this increase the image size?

**Comment by sumo-drosiek** [comment]
@kkujawa-sumo please re-review as I changed it slightly :sweat_smile:

File: Dockerfile [hunk 1]
```diff
@@ -10,16 +10,26 @@ RUN apk --update add ca-certificates
 FROM alpine:3.16.0 as directories
 RUN mkdir /etc/otel/
 
+FROM debian:11.3 as systemd
+RUN apt update && apt install -y systemd
+# prepare package with journald and it's dependencies
+# h stands for dereference of symbolic links
+RUN tar czhf journalctl.tar.gz /bin/journalctl $(ldd /bin/journalctl | grep -oP "\/.*? ")
+# extract package to /output so it can be take as base for scratch image
+RUN mkdir /output && tar xf /journalctl.tar.gz --directory /output
```
Comment (swiatekm-sumo) [inline Dockerfile:19]: Why are we compressing these files, if we immediately decompress them in the same build stage?

Comment (sumo-drosiek) [reply Dockerfile:19]: this is copying only required files while preserving paths and meantime skipping symlinks. pretty easy to explain :joy:

Comment (swiatekm-sumo) [reply Dockerfile:19]: Maybe at least leave a comment so it's clear why this exists, if not how it works exactly?

**Review by swiatekm-sumo — APPROVED** [review]

**Pull request merged**